### PR TITLE
BankID OIDC flow 

### PR DESCRIPTION
**Pull request by k-irishina**
Added ~~authorizationURL~~ authenticationURL tokenId to the notification towards merchant we'll send after registering permissions for enrolment.

### CODE REVIEW

File: openapi/integrator/wallet/server.yaml
Comment (thxmasj) [inline openapi/integrator/wallet/server.yaml:96]: This does not necessarily need to be used only for _HTTP redirect binding_ only, so we could rephrase to something like this:

> Endpoint used by the authentication provider to provide the authentication response.

File: openapi/integrator/wallet/server.yaml
Comment (thxmasj) [inline openapi/integrator/wallet/server.yaml:160]: oauth2AuthorisationCode

File: openapi/integrator/wallet/server.yaml
Comment (thxmasj) [inline openapi/integrator/wallet/server.yaml:166]: oauth2State

File: openapi/integrator/wallet/server.yaml
Comment (thxmasj) [inline openapi/integrator/wallet/server.yaml:172]: oauth2Error

File: openapi/integrator/wallet/server.yaml
Comment (thxmasj) [inline openapi/integrator/wallet/server.yaml:178]: oauth2ErrorDescription

**Review by thxmasj — CHANGES_REQUESTED** [review]

File: openapi/integrator/wallet/client.yaml
Comment (k-irishina) [reply openapi/integrator/wallet/client.yaml:261]: Then we're not aligned with oauth2 spec, but ok. :-)

**Review by thxmasj — APPROVED** [review]
Remember to not merge until just before we merge the platform.

**Pull request merged**
